samples/HttpRequestFeatures/README.md

```diff
@@ -1 +1,7 @@
 # About this sample
+This sample extracts HTTP request features and adds them as context feature to a Rank requests that are sent to an Azure Personalizer instance.
+
+# Run sample
+1. Create a Personalizer instance in the Azure portal.
+2. Fill in the `ApiKey` and `ServiceEndpoint` strings in appsettings.json. These can be found in your Cognitive Services Quick start tab in the Azure portal.
```

Comment:
>Fill in the `ApiKey` and `ServiceEndpoint` strings in appsettings.json [](start = 3, length = 70)

Setting API Keys in the appsettings.json is unsecure and a possible security problem during commits. I suggest sticking with the environment variable pattern in PersonalizerExample or better yet follow the pattern for `dotnet user-secrets` used in demos\PersonalizerDemo\README.md

Reply:
Updated both README and code